scripts/apps/contacts/helpers.js

```diff
@@ -44,7 +44,7 @@ export function renderArea(area, itemProps, props) {
             return fields[field](itemProps);
         }
 
-        console.warn('missing field in list: ' + field);
+        // console.warn('missing field in list: ' + field);
```

Comment:
I also hate this warning because I get it every day 😁, should we just remove it?